AI/vllm-deployment/vllm-service.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: v1
 kind: Service
 metadata:
   name: vllm-service
+  namespace: vllm-example
```

Comment:
Suggest omitting the namespace from resource manifests and specify it at deploy time.